openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/AppRepository.java

```diff
@@ -230,6 +233,26 @@ public EntityUpdater getUpdater(App original, App updated, Operation operation)
     return new AppRepository.AppUpdater(original, updated, operation);
   }
 
+  @Override
+  public void restorePatchAttributes(App original, App updated) {
+    super.restorePatchAttributes(original, updated);
+
+    // Check app configuration for search indexing application
+    // if searchIndexMappingLanguage is updated, check if plugin is present
+    if ("SearchIndexingApplication".equalsIgnoreCase(updated.getFullyQualifiedName())) {
```

Comment:
any reason we are looking based on specific application. We don't want to introduce application specific conditions into repository